src/navigation/navigation.tsx

```diff
@@ -395,6 +396,22 @@ const QRStack = (): JSX.Element => {
           header: (props: NativeStackHeaderProps) => <SSIHeaderBar {...props} />,
         }}
       />
+      <Stack.Screen
+        name={ScreenRoutesEnum.LOCK}
+        component={SSILockScreen}
+        initialParams={{onVerificationSuccess: onLogin}}
```

Comment:
this initialParams can be removed. since this is a screen in another stack being used for only entering your pincode

Reply:
changed